packages/ti_eset/data_stream/botnet/elasticsearch/ilm/default_policy.json

```
@@ -0,0 +1,23 @@
+{
+    "policy": {
+        "phases": {
+            "delete": {
+                "actions": {
+                    "delete": {}
+                },
+                "min_age": "3d"
```

```suggestion
                "min_age": "7d"
```
As per the README doc. Same for other datastreams that have `7d` in README doc.

---

Updated